draft-ietf-rtcweb-jsep.xml

```diff
@@ -766,9 +767,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 even global level.</t>
           </list>
           </t>
-          <t>Although it can be overridden by local policy, the default ICE
-          candidate policy MUST be set to allow all candidates, as this
-          minimizes use of application STUN/TURN server resources.</t>
+          <t>The default ICE candidate policy MUST be set to allow all
+          candidates, as this is generally the desired policy, and also
```

Comment:
We use "all candidates" elsewhere, so I think this tweak would be insufficient. But perhaps I could speak to the exact meaning of "all candidates" on line 754, and allude to the just-mentioned browser filtering.